src/components/Feed/Feed.js

```diff
@@ -3,12 +3,42 @@ import { Route } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import './Feed.css';
-
-// import Post from '../../components/Post/Post';
+import Post from '../../components/Post/Post';
+import { user, posts } from '../../Data/mockData';
 
 const Feed = () => {
+  const [currentUserPosts, setCurrentUserPosts] = useState([])
+  const [allPosts, setAllPosts] = useState([])
+
+  useEffect(() => {
+    setCurrentUserPosts(user.posts)
+    setAllPosts(posts)
+    // console.log('currentUserPosts', currentUserPosts)
+    // console.log('allPosts', allPosts)
```

Comment:
Not sure if you deleted these already.

Reply:
Thanks for the reminder, I will remove those before this gets merged!